src/main/java/com/openlattice/controllers/OrganizationsController.java

```diff
@@ -321,8 +295,7 @@ public Void refreshDataChanges(
         // the person requesting refresh should be the owner of the organization
         ensureOwner( organizationId );
```

Comment:
If we check owner access in this call, shouldn't we also do for others/delete it from here?